src/generate/renders/tests/ScenarioTool.js

```diff
@@ -0,0 +1,978 @@
+"use strict";
```

Comment:
is this file miss commited? seems ScenarioTool.ts is enough

Reply:
oh yes. let me delete it

Reply:
removed